mapsforge-map/src/main/java/org/mapsforge/map/layer/renderer/StandardRenderer.java

```diff
@@ -17,7 +17,11 @@
  */
 package org.mapsforge.map.layer.renderer;
 
-import org.mapsforge.core.graphics.*;
+import org.mapsforge.core.graphics.Bitmap;
+import org.mapsforge.core.graphics.Display;
+import org.mapsforge.core.graphics.GraphicFactory;
+import org.mapsforge.core.graphics.Paint;
+import org.mapsforge.core.graphics.Position;
```

Comment:
@menion
Can you revert this optimization of imports?
(as the code is unchanged to have clean history)